test/tests_dag.py

```diff
@@ -612,6 +612,73 @@ def test_dag_modelrun_financialNet(env):
             ret = con.execute_command("EXISTS {}".format(result_tensor_keyname))
             env.assertEqual(ret, 1)
 
+
+def test_dag_modelrun_financialNet_autobatch(env):
+    if not TEST_TF:
+        return
+    con = env.getConnection()
+
+    model_pb, creditcard_transactions, creditcard_referencedata = load_creditcardfraud_data(
+        env)
+    model_name = 'financialNet{{hhh}}'
+
+    ret = con.execute_command('AI.MODELSET', model_name, 'TF', 'CPU',
+                              'BATCHSIZE', 2, 'MINBATCHSIZE', 2,
+                              'INPUTS', 'transaction', 'reference', 'OUTPUTS', 'output',
+                              'BLOB', model_pb)
+    env.assertEqual(ret, b'OK')
+
+    for tensor_number in range(1,MAX_TRANSACTIONS):
+        for repetition in range(1,10):
+            reference_tensor = creditcard_referencedata[tensor_number]
+            transaction_tensor = creditcard_transactions[tensor_number]
+            result_tensor_keyname = 'resultTensor{{hhh}}{}'.format(tensor_number)
+            reference_tensor_keyname = 'referenceTensor{{hhh}}{}'.format(tensor_number)
+            transaction_tensor_keyname = 'transactionTensor{{hhh}}{}'.format(tensor_number)
+
+            ret = con.execute_command('AI.TENSORSET', reference_tensor_keyname,
+                                    'FLOAT', 1, 256,
+                                    'BLOB', reference_tensor.tobytes())
+            env.assertEqual(ret, b'OK')
+            ret = con.execute_command("EXISTS {}".format(reference_tensor_keyname))
+            env.assertEqual(ret, 1)
+
+            def run():
+                con = env.getConnection()
+                ret = con.execute_command(
+                    'AI.DAGRUN', 'LOAD', '1', reference_tensor_keyname, '|>',
+                    'AI.TENSORSET', transaction_tensor_keyname, 'FLOAT', 1, 30,'BLOB', transaction_tensor.tobytes(), '|>',
+                    'AI.MODELRUN', model_name,
+                                'INPUTS', transaction_tensor_keyname, reference_tensor_keyname,
+                                'OUTPUTS', result_tensor_keyname
+                )
+                ensureSlaveSynced(con, env)
+
+            t = threading.Thread(target=run)
+            t.start()
```

Comment:
The thread is used to trigger concurrent requests towards RedisAI, since `DAGRUN` is blocking so requests would be sequential. There's no `stop` method in `Thread`.